app/views/posts/_form.html.erb

```diff
@@ -17,12 +17,17 @@
   </div>
   
   <div class="field">
-    <%= f.file_field :picture %> 
```

Comment:
Powinny działać (dodawanie obrazków, bez formatowania)
  ----- Original Message ----- 
  From: Anna Ślimak 
  To: kraryk/test_blog 
  Cc: kraryk 
  Sent: Tuesday, January 26, 2016 3:18 PM
  Subject: Re: [test_blog] BŁąd przy wywoływaniu @post.tags (#3)


  In app/views/posts/_form.html.erb:

> @@ -17,12 +17,17 @@
>    </div>
>    
>    <div class="field">
> -    <%= f.file_field :picture %> 
co z obrazkami?

  —
  Reply to this email directly or view it on GitHub.